src/main/java/duke/commands/DoneCommand.java

```diff
@@ -0,0 +1,39 @@
+package duke.commands;
+
+import duke.system.Storage;
+import duke.system.DukeException;
+import duke.system.Ui;
+import duke.system.State;
+import duke.tasks.Task;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Marks a task in the list as done
+ */
+public class DoneCommand extends Command{
+    public DoneCommand(String input, ArrayList<Task> tasks, State state) {
+        super(input, tasks, state);
+    }
+
+    public  void execute() {
```

Comment:
All public methods of a class must be provided with header comments. (This comment is applicable for all the public methods inside various classes of your project)